ppr-ui/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "serve": "vite dev",
+    "serve": "vite preview --port 8080",
```

Comment:
@dimak1 `npm run dev` for local dev 👍 
`serve` for viewing production ready build locally, following a local build.

Reply:
`npm run dev` is the one for local dev.

Reply:
Since you've changed this file, I recommend updating the patch version on line 3.

Reply:
Done!